### DISCUSSION TITLE
Haven't recompiled in a while, but now getting error.

### DISCUSSION THREAD

**Discussion by crzykidd** [emporia-vue-local/esphome#320] [Q&A]
I haven't recompiled or change a config in a while maybe 9 months.   However I had a new circuit added so wanted to get that setup.   When I try to install or validate in ESPHome I get this following message.   I am sure this URL is correct, other items in my devices use git and I can update them.     Any help would be greatly appreciated: 

`INFO ESPHome 2024.10.3
INFO Reading configuration /config/vue2-inverter-panel.yaml...
Failed config

external_components: [source /config/vue2-inverter-panel.yaml:66]
  - 
    fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree.
    Use '--' to separate paths from revisions, like this:
    'git <command> [<revision>...] -- [<file>...]'.
    source: github://emporia-vue-local/esphome@dev
    components: 
      - emporia_vue`

**Comment by flaviut** [comment]
Try "Clean Build Files".
